activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+*   Enforce fresh ETag header after a collection's contents change by adding
+    ActiveRecord::Relation#cache_key_with_version. This method will be used by
+    ActionController::ConditionalGet to ensure that when collection cache versioning
+    is enabled, requests using ConditionalGet don't return the same ETag header
+    after a collection is modified. Fixes #38078. *Aaron Lipman*
```

Comment:
You name needs to be in a different line

Reply:
Rebased accordingly - but the Rails guide seems to indicate that a single paragraph is appropriate:

> Your name can be added directly after the last word if there are no code examples or multiple paragraphs. Otherwise, it's best to make a new paragraph.

https://edgeguides.rubyonrails.org/contributing_to_ruby_on_rails.html#updating-the-changelog